test/tests/transfers/multiapp_conservative_transfer/master_nearest_point.i

```diff
@@ -0,0 +1,134 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmax = 1
+  ymax = 1
+  nx = 10
+  ny = 10
+[]
+
+[MeshModifiers]
+  [block1]
+    type = SubdomainBoundingBox
+    block_id = 1
+    bottom_left = '0.5 0 0'
+    top_right = '1 1 0'
+  []
+[]
+
+[Variables]
+  [power_density]
+  []
+[]
+
+[Functions]
+  [pwr_func]
+    type = ParsedFunction
+    value = '1e3*x*(1-x)+5e2'
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = power_density
+  []
+
+  [coupledforce]
+    type = BodyForce
+    variable = power_density
+    function = pwr_func
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = power_density
+    boundary = left
+    value = 0
+  []
+  [right]
+    type = DirichletBC
+    variable = power_density
+    boundary = right
+    value = 1e3
+  []
+[]
+
+[AuxVariables]
+  [from_sub]
+  []
+[]
+
+[Postprocessors]
+  [from_nearest_point]
+    type = NearestPointIntegralVariablePostprocessor
```

Comment:
It looks like the value of the `NearestPointIntegralVariablePostprocessor` pps are not printed to screen (or in the CSV file with `csv=true`). It would be useful to have it, especially to check that the pps indeed match.

Reply:
This still seems to be a problem

Reply:
I do not get it here. What do you mean by "still a problem"? Could you tell what you want?

Reply:
NearestPointIntegralVariablePostprocessor now is a standard vector post processor, and its values can be printed to a `csv` file if you you have `"csv=true"`

Reply:
For some reason, the value of this postprocessor never appears to my screen:

```
Running App: main
Framework Information:
MOOSE Version:           git commit 8f529ee8dc on 2019-05-17
LibMesh Version:         c4c9fd5450489486fd5f7baf2ec9a8ac0d47fc99
PETSc Version:           3.9.4
Current Time:            Fri May 17 12:10:12 2019
Executable Timestamp:    Fri May 17 11:46:44 2019

Parallelism:
  Num Processors:          1
  Num Threads:             1

Mesh: 
  Parallel Type:           replicated
  Mesh Dimension:          2
  Spatial Dimension:       2
  Nodes:                   
    Total:                 121
    Local:                 121
  Elems:                   
    Total:                 100
    Local:                 100
  Num Subdomains:          2
  Num Partitions:          1

Nonlinear System:
  Num DOFs:                121
  Num Local DOFs:          121
  Variables:               "power_density" 
  Finite Element Types:    "LAGRANGE" 
  Approximation Orders:    "FIRST" 

Auxiliary System:
  Num DOFs:                121
  Num Local DOFs:          121
  Variables:               "from_sub" 
  Finite Element Types:    "LAGRANGE" 
  Approximation Orders:    "FIRST" 

Execution Information:
  Executioner:             Steady
  Solver Mode:             Preconditioned JFNK
  Preconditioner:          hypre boomeramg 

Initializing All MultiApps
Creating MultiApp sub of type MooseTestApp of level 1 and number 0:
sub0: 
Creating MultiApp sub of type MooseTestApp of level 1 and number 1:
sub1: 
sub0: 
sub0: Running App: sub0
sub0: Parallelism:
sub0:   Num Processors:          1
sub0:   Num Threads:             1
sub0: 
sub0: Mesh: 
sub0:   Parallel Type:           replicated
sub0:   Mesh Dimension:          2
sub0:   Spatial Dimension:       2
sub0:   Nodes:                   
sub0:     Total:                 121
sub0:     Local:                 121
sub0:   Elems:                   
sub0:     Total:                 100
sub0:     Local:                 100
sub0:   Num Subdomains:          2
sub0:   Num Partitions:          1
sub0: 
sub0: Nonlinear System:
sub0:   Num DOFs:                100
sub0:   Num Local DOFs:          100
sub0:   Variables:               "sink" 
sub0:   Finite Element Types:    "MONOMIAL" 
sub0:   Approximation Orders:    "CONSTANT" 
sub0: 
sub0: Auxiliary System:
sub0:   Num DOFs:                21
sub0:   Num Local DOFs:          21
sub0:   Variables:               "from_master" 
sub0:   Finite Element Types:    "LAGRANGE" 
sub0:   Approximation Orders:    "FIRST" 
sub0: 
sub0: Execution Information:
sub0:   Executioner:             Steady
sub0:   Solver Mode:             Preconditioned JFNK
sub0:   Preconditioner:          hypre boomeramg 
sub0: 
sub1: 
sub1: Running App: sub1
sub1: Parallelism:
sub1:   Num Processors:          1
sub1:   Num Threads:             1
sub1: 
sub1: Mesh: 
sub1:   Parallel Type:           replicated
sub1:   Mesh Dimension:          2
sub1:   Spatial Dimension:       2
sub1:   Nodes:                   
sub1:     Total:                 121
sub1:     Local:                 121
sub1:   Elems:                   
sub1:     Total:                 100
sub1:     Local:                 100
sub1:   Num Subdomains:          2
sub1:   Num Partitions:          1
sub1: 
sub1: Nonlinear System:
sub1:   Num DOFs:                100
sub1:   Num Local DOFs:          100
sub1:   Variables:               "sink" 
sub1:   Finite Element Types:    "MONOMIAL" 
sub1:   Approximation Orders:    "CONSTANT" 
sub1: 
sub1: Auxiliary System:
sub1:   Num DOFs:                21
sub1:   Num Local DOFs:          21
sub1:   Variables:               "from_master" 
sub1:   Finite Element Types:    "LAGRANGE" 
sub1:   Approximation Orders:    "FIRST" 
sub1: 
sub1: Execution Information:
sub1:   Executioner:             Steady
sub1:   Solver Mode:             Preconditioned JFNK
sub1:   Preconditioner:          hypre boomeramg 
sub1: 
Finished Initializing All MultiApps

Backing Up MultiApps on TIMESTEP_END
Beginning backing up MultiApp sub
Finished backing up MultiApp sub
Finished Backing Up MultiApps on TIMESTEP_END

 0 Nonlinear |R| = 3.317228e+03
      0 Linear |R| = 3.317228e+03
      1 Linear |R| = 8.653417e+01
      2 Linear |R| = 2.749651e+00
      3 Linear |R| = 8.942078e-02
      4 Linear |R| = 2.238424e-03
 1 Nonlinear |R| = 6.375386e-02
      0 Linear |R| = 6.375386e-02
      1 Linear |R| = 1.596349e-03
      2 Linear |R| = 4.511555e-05
      3 Linear |R| = 1.604996e-06
      4 Linear |R| = 4.238846e-08
 2 Nonlinear |R| = 1.147928e-07
 Solve Converged!

Starting Transfers on TIMESTEP_END To MultiApps
Beginning MeshFunctionTransfer to_sub
Finished MeshFunctionTransfer to_sub
Beginning Conservative transfers to_sub
Finished Conservative transfers to_sub
Transfers on TIMESTEP_END Are Finished


Executing MultiApps on TIMESTEP_END
sub0: 
sub0: Postprocessor Values:
sub0: +----------------+----------------+----------------+
sub0: | time           | from_master_pp | sink           |
sub0: +----------------+----------------+----------------+
sub0: |   0.000000e+00 |   1.811928e+01 |   0.000000e+00 |
sub0: +----------------+----------------+----------------+
sub0: 
sub0:  0 Nonlinear |R| = 3.467168e+00
sub0:       0 Linear |R| = 3.467168e+00
sub0:       1 Linear |R| = 9.331840e-06
sub0:  1 Nonlinear |R| = 2.712319e-06
sub0:       0 Linear |R| = 2.712319e-06
sub0:       1 Linear |R| = 3.594458e-13
sub0:  2 Nonlinear |R| = 3.842817e-13
sub0:  Solve Converged!
sub0: 
sub0: Postprocessor Values:
sub0: +----------------+----------------+----------------+
sub0: | time           | from_master_pp | sink           |
sub0: +----------------+----------------+----------------+
sub0: |   0.000000e+00 |   1.811928e+01 |   0.000000e+00 |
sub0: |   1.000000e+00 |   1.538888e+02 |   4.663411e+00 |
sub0: +----------------+----------------+----------------+
sub0: 
sub1: 
sub1: Postprocessor Values:
sub1: +----------------+----------------+----------------+
sub1: | time           | from_master_pp | sink           |
sub1: +----------------+----------------+----------------+
sub1: |   0.000000e+00 |   4.691928e+01 |   0.000000e+00 |
sub1: +----------------+----------------+----------------+
sub1: 
sub1:  0 Nonlinear |R| = 3.467168e+00
sub1:       0 Linear |R| = 3.467168e+00
sub1:       1 Linear |R| = 9.331840e-06
sub1:  1 Nonlinear |R| = 2.712319e-06
sub1:       0 Linear |R| = 2.712319e-06
sub1:       1 Linear |R| = 3.594458e-13
sub1:  2 Nonlinear |R| = 3.842817e-13
sub1:  Solve Converged!
sub1: 
sub1: Postprocessor Values:
sub1: +----------------+----------------+----------------+
sub1: | time           | from_master_pp | sink           |
sub1: +----------------+----------------+----------------+
sub1: |   0.000000e+00 |   4.691928e+01 |   0.000000e+00 |
sub1: |   1.000000e+00 |   4.038887e+02 |   4.663411e+00 |
sub1: +----------------+----------------+----------------+
sub1: 
Finished Executing MultiApps on TIMESTEP_END


Starting Transfers on TIMESTEP_END From MultiApps
Beginning MeshFunctionTransfer from_sub
Finished MeshFunctionTransfer from_sub
Beginning Conservative transfers from_sub
Finished Conservative transfers from_sub
Transfers on TIMESTEP_END Are Finished


sub1: 
sub0: 
```

Is that normal?

Reply:
It makes it hard to check that the transfers are indeed conservative